docs/faq.html

```diff
@@ -75,119 +73,158 @@ <h1>About Schema.org</h1>
-
-  <p class="question">Q:
-    <a name="3"
-       id="3"></a>Is schema.org a standards body like the W3C or IETF ?</p>
-  <p>No. <ol><li>Schema.org is a collaboration, in the spirit of sitemaps.org, between Bing, Google, Yahoo! and Yandex to make it easier for webmasters to provide us with data so that we may better direct users to their sites.</li><li>Schema.org is not a formal standards body. Schema.org is simply a site where we document the schemas that several major search engines will support. </li><li>Schema.org is a collaboration between Google, Microsoft, Yahoo! and Yandex - large search engines who will use this marked-up data from web pages. Other sites - not necessarily search engines - might later join.</li></ol></p>
-
-  <p class="question">Q:
-    <a name="4"
-       id="4"></a>How does schema.org relate to Facebook Open Graph?</p>
-  <p>Facebook Open Graph serves its purpose well, but it doesn't provide the detailed information search engines need to improve the user experience. A single web page may have many components, and it may talk about more than one thing. If search engines understand the various components of a page, we can improve our presentation of the data. Even if you mark up your content using the Facebook Open Graph protocol, schema.org provides a mechanism for providing more detail about particular entities on the page. <br>For example, a page about a band could include any or all of the following:<ol><li>A list of albums</li><li>A price for each album</li><li>A list of songs for each album, along with a link to hear samples of each song</li><li>A list of upcoming shows</li><li>Bios of the band members</li></ol></p>
-
-  <p class="question">Q:
+<div itemscope itemprop="mainEntity" itemtype="https://schema.org/Question">
+  <p itemprop="name" class="question">Q:
+    <a name="0" id="0"></a>What is the purpose of schema.org?</p>
+  <div itemscope itemprop="acceptedAnswer" itemtype="https://schema.org/Answer">
+  <p itemprop="name">Schema.org is a joint effort, in the spirit of sitemaps.org, to improve the web by creating a structured data markup schema supported by major search engines.  On-page markup helps search engines understand the information on web pages and provide richer search results. A shared markup vocabulary makes easier for webmasters to decide on a markup schema and get the maximum benefit for their efforts. Search engines want to make it easier for people to find relevant information on the web. Markup can also enable new tools and applications that make use of the structure.</p>
```

Comment:
should this not be 'itemprop="answerExplanation"' instead?

Reply:
I followed the pattern described here: https://developers.google.com/search/docs/appearance/structured-data/faqpage